qa-core/src/config/internal-api/TestConfiguration/InternalAPIClient.ts

```diff
@@ -49,14 +49,17 @@ class InternalAPIClient {
       // @ts-ignore
       const response = await fetch(`https://${process.env.TENANT_ID}.${this.host}${url}`, requestOptions)
 
-      if (response.status == 404 || response.status == 500) {
+      if (
+        response.status == 404 ||
+        response.status == 500 ||
+        response.status == 403
+      ) {
         console.error("Error in apiCall")
         console.error("Response:")
         console.error(response)
+        const json = await response.json()
         console.error("Response body:")
-        console.error(response.body)
-        console.error("Request body:")
-        console.error(requestOptions.body)
+        console.error(json)
```

Comment:
`console.error` here doesn't seem right - might have been left in from debugging just.

Reply:
@shogunpurple `console.log` doesn't work in the pipeline and its useful to have this information in case of failures. However and as you pointed in another comment, these logs are done poorly. Will change.